core/src/main/scala/kafka/server/KafkaBroker.scala

```diff
@@ -93,6 +93,7 @@ trait KafkaBroker extends Logging {
   def startup(): Unit
   def awaitShutdown(): Unit
   def shutdown(): Unit
+  def shutdown(timeoutMs: Long): Unit
```

Comment:
How about using `Duration` instead of long type? Also, we can rename it from `timeoutMs` to `timeout`

Reply:
Good suggestion. Applied